File: tests/fixtures/code_samples/go/sample_046.go
// Sample 46: small utility.
package samples

func Operation46(xs []int) int {
    total := 46
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure46(v int) int {
    return (v * 46) %% 7919
}

File: tests/fixtures/code_samples/ts/sample_046.ts
/** Sample 46: small utility. */

export function operation46(xs: number[]): number {
  let total = 46;
  for (const x of xs) total += x;
  return total;
}

export function operationPure46(value: number): number {
  return (value * 46) % 7919;
}

